routers/api/v1/router.go

```diff
@@ -40,9 +51,14 @@ func (r *apiV1Router) RegisterRoutes(routerGroup *gin.RouterGroup) {
 
 	usersGroup := routerGroup.Group("/users")
 
-	usersGroup.GET("/", r.GetUsers)
+	isAtLeastApplicant := r.AuthLevelVerifierFactory(authlevels.Applicant)
```

Comment:
Small thing, but could it be `r` be renamed `router`? It might make it a bit clearer

Reply:
Its a golang convention to call method receivers with a single character